core/arch/arm/plat-imx/registers/imx6-regs.h

```diff
@@ -0,0 +1,42 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2017-2018 NXP
+ *
```

Comment:
minor: remove empty line

Reply:
Shouldn't there be "Copyright (c) 2016, Wind River Systems." here? Last contrib: 
https://github.com/OP-TEE/optee_os/commit/ea850f1fe8b9354a97351939656f889d0ff338a4.